### PR TITLE
supporter plus price rise

### PR DESCRIPTION
**Pull request by rBangay**
## What does this change?

Update the supporter plus min amount threshold prices to align with an inbound checkout price rise

## Trello card
https://trello.com/c/yp41n8NB/353-supporter-plus-price-rise-upsell

## Do not merge until instructed to by our P&P team overlords 👑

### CODE REVIEW

**Comment by johnduffell** [comment]
where it the source data for this? is there a card or spreadsheet for it?

**Comment by rBangay** [comment]
> where it the source data for this? is there a card or spreadsheet for it?

Yep, theres a link to the spreadsheet in the trello card:

https://trello.com/c/yp41n8NB/353-supporter-plus-price-rise-upsell

I'll add the card to the pr descriotion as well

File: client/utilities/pricingConfig/suggestedAmounts.ts [hunk 1]
```diff
@@ -65,27 +65,27 @@ const suggestedAmountsLookup: SuggestedAmountsLookup = {
...
 function getGbpMonthly(amount: number) {
 	if (amount <= 3) {
-		return [5, 7, 10];
+		return [5, 7, 12];
 	}
 	if (amount <= 8) {
 		return [10, 12, 15];
 	}
-	if (amount <= 9) {
+	if (amount === 9) {
```
Comment (rupertbates) [inline client/utilities/pricingConfig/suggestedAmounts.ts:73]: Why has this line changed?

**Review by rupertbates — APPROVED** [review]
One question about why <= has become === in a number of places.

**Comment by rBangay** [comment]
> One question about why <= has become === in a number of places.

Becuase it marries with the source data in the spreadsheet ... (functionaly it makes no difference with the lower conditions above in the code catching the < cases). it's more explicit now

**Review by paul-daniel-dempsey — APPROVED** [review]
Looks good - amount changes look exactly as in price-rise spreadsheet

**Pull request merged**